src/content/docs/en/pages/secure-journey/edit-edge-firewall/protect-your-domain.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Protect your domain
+description: >-
+  Learn how to configure an edge firewall to protect your domain.
+meta_tags: 'secure, edge firewall, configuration, settings, domains'
+namespace: docs_guides_secure_protect_your_domain
+permalink: /documentation/products/guides/secure/edge-firewall-protect-your-domain/
+menu_namespace: secureMenu
+---
+
+import Tabs from '~/components/tabs/Tabs'
+import InterfaceNote from '~/includes/snippets/InterfaceNote/en/snippet.mdx'
+
+<InterfaceNote />
+
+Once you create an edge firewall with Azion, you can modify a few configurations. 
+
+In this guide, you'll learn how to select an edge firewall to protect a particular domain.
+
+<Tabs client:visible>
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+On the Console, you must now configure your domain so your edge firewall protects it.
+
+1. On the **Products menu**, select **Domains**.
+2. Click on the domain you want to protect with your edge firewall.
+3. In the **Settings** section, click the `Edge Firewall` selector and choose the edge firewall.
+4. Click the **Save** button.
+
+Done. Now the selected edge firewall will protect the domain you indicated.
+</Fragment>
+
+<Fragment slot="panel.rtm">
+In case you are using RTM, this process is done while [configuring or editing the edge firewall](/en/documentation/products/guides/secure/edge-firewall-configure-main-settings/#domains-1).
```

Comment:
```suggestion
In case you're using Real-Time Manager (RTM), this process is done while [configuring or editing the edge firewall](/en/documentation/products/guides/secure/edge-firewall-configure-main-settings/#domains-1).
```